modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java

```diff
@@ -643,12 +650,17 @@ protected void updateLogLevels(ActionRequest actionRequest)
 
 				Log4JUtil.setLevel(loggerName, priority, true);
 
-				logLevels.add(new LogLevel(loggerName, priority));
+				log4JLevelConfigs.add(
+					new Log4JUtil.Log4JLevelConfig(loggerName, priority, true));
 			}
 		}
 
-		if (!logLevels.isEmpty()) {
-			_executeOnCluster(_updateLogLevelsMethodKey, logLevels);
+		if (ClusterMasterExecutorUtil.isMaster()) {
+			_notifySlaves(
```

Comment:
@marianoalvarosaiz  I can understand that you trigger a cluster event to update cluster what's changed, but I do not understand why you notify slaves with all current logic4j configs retrieved by "Log4JUtil.getLog4JLevelConfigs()".

Reply:
@tinatian https://github.com/shuyangzhou/liferay-portal/pull/10494#issuecomment-994989752 Shuyang told that since it's a not often operation we call broadcast all information.

Reply:
@marianoalvarosaiz OK, I see...